h/services/__init__.py

```diff
@@ -7,102 +7,132 @@
 
 
 def includeme(config):  # pragma: no cover
-    config.register_service_factory(".annotation_json.factory", name="annotation_json")
+    # Annotation related services
```

Comment:
Not a fan of these sorting markers (and the sorting in general)

This is not the type of file I read top to bottom, just search for the service I'm interested in .

Not that I would ask for you to now randomly sort the services.

Reply:
These are pretty much just sorted alphabetically with occasional markers to help the eye. By having a more consistent naming style this naturally groups related services.

I wasn't 100% about making this true, but it's closer. I might go back over this and make it 100% true.

Reply:
I've had another go and put it into strict alphabetical order. I was trying to not disrupt things too much, but honestly I should just have sorted it fully.

I've sorted these by the file path, so you can match them up more easily.

I also fixed one remaining relative import I missed.